src/evaluations/evaluator.py

```diff
@@ -198,7 +203,9 @@ def __init__(self, evaluation_config, sketch_estimator_config_list, run_name,
       assert isinstance(sketch_estimator_config, SketchEstimatorConfig), (
           'sketch_estimator_config_list does not contain all '
           'SketchEstimatorConfig objects')
-
+    assert isinstance(parallel_cores, int), (
+        'parallel_cores must be an int')
+    self.parallel_cores = parallel_cores if parallel_cores >= 1 else cpu_count()
```

Comment:
What happens if the user requests more than the maximum available number of cores?

Reply:
Ah this is a good point. If the user requests more than the maximum cores, then that number of processes are spawned and there will be at least 1 core with more than 1 process. It might be best to cap it to the maximum amount of processors because no performance gains will be had from spawning more than 1 process per core.

Reply:
Resolved in 27cfae7